avaframe/out1Peak/outPlotAllPeak.py

```diff
@@ -208,7 +251,7 @@ def plotAllFields(avaDir, inputDir, outDir, unit="", constrainData=True):
     if outDir.is_dir() is False:
         # create out dir if not already existing
         outDir.mkdir()
-    peakFiles = list(inputDir.glob("*.asc"))
+    peakFiles = list(inputDir.glob("*.asc")) + list(inputDir.glob("*.tif"))
```

Comment:
do we now allow peakfiles ot be .tif and do we also consider .tif as an output format for com1DFA result files?

Reply:
Yes. Whatever the format of the dem is...